rest-api/src/main/java/io/quarkus/qe/RepositoryResource.java

```diff
@@ -3,12 +3,7 @@
 import javax.inject.Inject;
 import javax.transaction.Transactional;
 import javax.validation.Valid;
-import javax.ws.rs.Consumes;
-import javax.ws.rs.GET;
-import javax.ws.rs.POST;
-import javax.ws.rs.Path;
-import javax.ws.rs.PathParam;
-import javax.ws.rs.Produces;
+import javax.ws.rs.*;
```

Comment:
Did you this on purpose? My IDE (which is using the formatter in Quarkusio) is undoing this change when I save the file :/